arangod/Aql/IndexExecutor.h

```diff
@@ -216,14 +214,10 @@ class IndexExecutor {
   inline arangodb::OperationCursor* getCursor(size_t pos) {
     return _cursors[pos].get();
   }
-  std::vector<std::unique_ptr<OperationCursor>>& getCursors() {
-    return _cursors;
-  }
 
   void setIndexesExhausted(bool flag) { _indexesExhausted = flag; }
   bool getIndexesExhausted() { return _indexesExhausted; }
 
-  void setLastIndex(bool flag) { _isLastIndex = flag; }
   bool isLastIndex() { return _isLastIndex; }
```

Comment:
@hkernbach You're right in that `setLastIndex` is never called, so removing it doesn't change anything. I wonder, however: It looks to me like `_isLastIndex` is only set to `false` in the constructor and never changed. But `isLastIndex()` is used. Can you help to clear this up? Either this is currently broken because a `setLastIndex` call is missing, or the case distinction can be removed.

Reply:
(note that this does not have to be fixed in this PR, but we must check this)

Reply:
Nice! There was indeed something missing. Fixed.